BUG: make test fails when building from binaries using v22.1.0

When building from binaries using v22.1.0 release [here](https://github.com/ConsenSys/quorum/releases/tag/v22.1.0) make test fails - source documentation for instructions [here]( https://consensys.net/docs/goquorum/en/latest/deploy/install/binaries/)

**Abreviated logs here**
```
--- FAIL: TestUPNP_DDWRT (2.16s)
    natupnp_test.go:199: HTTPU request M-SEARCH *
    natupnp_test.go:199: HTTPU request M-SEARCH *
    natupnp_test.go:199: HTTPU request M-SEARCH *
    natupnp_test.go:199: HTTPU request M-SEARCH *
    natupnp_test.go:199: HTTPU request M-SEARCH *
    natupnp_test.go:199: HTTPU request M-SEARCH *
    natupnp_test.go:212: HTTP request "GET /InternetGatewayDevice.xml" --> 200
    natupnp_test.go:212: HTTP request "POST /control?WANIPConnection" --> 200
    natupnp_test.go:169: upnp.service mismatch: got "IGDv2-IP2", want "IGDv1-IP1"
    natupnp_test.go:173: upnp.dev.URLBaseStr mismatch: got "http://192.168.86.1:5000/rootDesc.xml", want "http://127.0.0.1:55356/InternetGatewayDevice.xml"
FAIL
FAIL	github.com/ethereum/go-ethereum/p2p/nat	3.038s

....

INFO [02-26|14:51:35.246] Persisted trie from memory database      nodes=1 size=144.00B time="16.458µs" gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.249] Loaded most recent local header          number=0 hash="62413f…e79654" td=0 age=52y10mo4w
INFO [02-26|14:51:35.249] Loaded most recent local full block      number=0 hash="62413f…e79654" td=0 age=52y10mo4w
INFO [02-26|14:51:35.249] Loaded most recent local fast block      number=0 hash="62413f…e79654" td=0 age=52y10mo4w
WARN [02-26|14:51:35.249] Failed to load snapshot, regenerating    err="missing or corrupted snapshot"
INFO [02-26|14:51:35.249] Rebuilding state snapshot 
INFO [02-26|14:51:35.249] Deleted state snapshot leftovers         kind=accounts wiped=0 elapsed="7.75µs"
INFO [02-26|14:51:35.249] Wiper running, state snapshotting paused accounts=0 slots=0 storage=0.00B elapsed="213.667µs"
INFO [02-26|14:51:35.250] Deleted state snapshot leftovers         kind=storage  wiped=0 elapsed="2.167µs"
INFO [02-26|14:51:35.250] Compacting snapshot account area  
INFO [02-26|14:51:35.250] Compacting snapshot storage area  
INFO [02-26|14:51:35.250] Compacted snapshot area in database      elapsed="57.5µs"
INFO [02-26|14:51:35.250] Resuming state snapshot generation       root="53e6df…804de7" accounts=0 slots=0 storage=0.00B elapsed="4.959µs"
INFO [02-26|14:51:35.251] Generated state snapshot                 accounts=1 slots=0 storage=43.00B elapsed="290.209µs"
INFO [02-26|14:51:35.255] Imported new chain segment               blocks=1 txs=1 mgas=0.055 elapsed=2.291ms     mgasps=24.183 number=1 hash="6f075f…f2da89" age=52y10mo4w dirty=958.00B
INFO [02-26|14:51:35.257] Persisted trie from memory database      nodes=1 size=144.00B time="2.333µs"  gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.258] Loaded most recent local header          number=0 hash="62413f…e79654" td=0 age=52y10mo4w
INFO [02-26|14:51:35.258] Loaded most recent local full block      number=0 hash="62413f…e79654" td=0 age=52y10mo4w
INFO [02-26|14:51:35.258] Loaded most recent local fast block      number=0 hash="62413f…e79654" td=0 age=52y10mo4w
WARN [02-26|14:51:35.258] Failed to load snapshot, regenerating    err="missing or corrupted snapshot"
INFO [02-26|14:51:35.258] Rebuilding state snapshot 
INFO [02-26|14:51:35.258] Deleted state snapshot leftovers         kind=accounts wiped=0 elapsed="3.208µs"
INFO [02-26|14:51:35.258] Deleted state snapshot leftovers         kind=storage  wiped=0 elapsed="7.916µs"
INFO [02-26|14:51:35.258] Compacting snapshot account area  
INFO [02-26|14:51:35.258] Compacting snapshot storage area  
INFO [02-26|14:51:35.258] Compacted snapshot area in database      elapsed="7.042µs"
INFO [02-26|14:51:35.258] Wiper running, state snapshotting paused accounts=0 slots=0 storage=0.00B  elapsed="119.792µs"
INFO [02-26|14:51:35.258] Resuming state snapshot generation       root="53e6df…804de7" accounts=0 slots=0 storage=0.00B  elapsed="1.458µs"
INFO [02-26|14:51:35.258] Generated state snapshot                 accounts=1 slots=0 storage=43.00B elapsed="18.125µs"
INFO [02-26|14:51:35.259] Imported new chain segment               blocks=1 txs=1 mgas=0.053 elapsed="246.166µs" mgasps=215.302 number=1 hash="51119d…45a8d7" age=52y10mo4w dirty=958.00B
INFO [02-26|14:51:35.260] Writing cached state to disk             block=1 hash="51119d…45a8d7" root="c00f9a…b1a0f1"
INFO [02-26|14:51:35.260] Persisted trie from memory database      nodes=4 size=574.00B time="14.333µs" gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.260] Writing snapshot state to disk           root="53e6df…804de7"
INFO [02-26|14:51:35.260] Persisted trie from memory database      nodes=0 size=0.00B   time=458ns      gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.260] Blockchain stopped 
INFO [02-26|14:51:35.260] Writing cached state to disk             block=1 hash="6f075f…f2da89" root="9b99e6…d5fef1"
INFO [02-26|14:51:35.260] Persisted trie from memory database      nodes=4 size=574.00B time="11.625µs" gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.260] Writing snapshot state to disk           root="53e6df…804de7"
INFO [02-26|14:51:35.260] Persisted trie from memory database      nodes=0 size=0.00B   time=292ns      gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.260] Blockchain stopped 
INFO [02-26|14:51:35.261] Persisted trie from memory database      nodes=1 size=144.00B time="2.5µs"    gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.262] Loaded most recent local header          number=0 hash="62413f…e79654" td=0 age=52y10mo4w
INFO [02-26|14:51:35.262] Loaded most recent local full block      number=0 hash="62413f…e79654" td=0 age=52y10mo4w
INFO [02-26|14:51:35.262] Loaded most recent local fast block      number=0 hash="62413f…e79654" td=0 age=52y10mo4w
WARN [02-26|14:51:35.262] Failed to load snapshot, regenerating    err="missing or corrupted snapshot"
INFO [02-26|14:51:35.262] Rebuilding state snapshot 
INFO [02-26|14:51:35.262] Deleted state snapshot leftovers         kind=accounts wiped=0 elapsed="2.209µs"
INFO [02-26|14:51:35.262] Deleted state snapshot leftovers         kind=storage  wiped=0 elapsed="1.042µs"
INFO [02-26|14:51:35.262] Compacting snapshot account area  
INFO [02-26|14:51:35.262] Compacting snapshot storage area  
INFO [02-26|14:51:35.262] Compacted snapshot area in database      elapsed="5.459µs"
INFO [02-26|14:51:35.262] Wiper running, state snapshotting paused accounts=0 slots=0 storage=0.00B  elapsed="70.625µs"
INFO [02-26|14:51:35.262] Resuming state snapshot generation       root="53e6df…804de7" accounts=0 slots=0 storage=0.00B  elapsed="1.292µs"
INFO [02-26|14:51:35.262] Generated state snapshot                 accounts=1 slots=0 storage=43.00B elapsed="17.875µs"
INFO [02-26|14:51:35.263] Imported new chain segment               blocks=1 txs=1 mgas=0.024 elapsed="359.25µs"  mgasps=67.864  number=1 hash="749482…1a3ca3" age=52y10mo4w dirty=692.00B
INFO [02-26|14:51:35.264] Writing cached state to disk             block=1 hash="749482…1a3ca3" root="e746ce…087d30"
INFO [02-26|14:51:35.264] Persisted trie from memory database      nodes=3 size=404.00B time="11.875µs" gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.264] Writing snapshot state to disk           root="53e6df…804de7"
INFO [02-26|14:51:35.264] Persisted trie from memory database      nodes=0 size=0.00B   time=417ns      gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [02-26|14:51:35.264] Blockchain stopped 
FAIL
FAIL	github.com/ethereum/go-ethereum/accounts/abi/bind	1.033s

...

FAIL
util.go:47: exit status 1
exit status 1
make: *** [test] Error 1
```

--- a/p2p/nat/natupnp_test.go
+++ b/p2p/nat/natupnp_test.go
@@ -21,6 +21,7 @@ import (
 	"io"
 	"net"
 	"net/http"
+	"os"
 	"runtime"
 	"strings"
 	"testing"
@@ -162,7 +163,11 @@ func TestUPNP_DDWRT(t *testing.T) {
 	// Attempt to discover the fake device.
 	discovered := discoverUPnP()
 	if discovered == nil {
-		t.Fatalf("not discovered")
+		if os.Getenv("CI") != "" {
+			t.Fatalf("not discovered")
+		} else {
+			t.Skipf("UPnP not discovered (known issue, see https://github.com/ethereum/go-ethereum/issues/21476)")
+		}
 	}
 	upnp, _ := discovered.(*upnp)
 	if upnp.service != "IGDv1-IP1" {

